packages/tests/src/update.ts

```diff
@@ -133,12 +133,12 @@ namespace OrmOperations {
         data.list = ['2', '3', '3']
         return data.id
       })
-      await database.set('temp2', {
+      await expect(database.set('temp2', {
         $or: [
           { id: magicIds },
           { timestamp: magicBorn },
         ],
-      }, { list: ['2', '3', '3'] })
+      }, { list: ['2', '3', '3'] })).to.eventually.deep.equal({ modified: 3 })
```

Comment:
`eventually` should be omitted here and everywhere?

Reply:
can't omit

Reply:
they are all promises

Reply:
Ah...